### PR TITLE
set explicit limit for Argo facet fields

### LINKED ISSUE #286: Number of objects in facets: match production or other solution
screenshot 2015-12-02 09 30 39

In Argo-stage, facets are limited to 10 objects, with no additional "more" links or ability to scroll or ctrl-f to find. If https://github.com/sul-dlss/argo/issues/224 is underdesigned or out of scope, we need to at least match the current production approach (just list 'em all).

### PR DESCRIPTION
**Pull request by mejackreed**
Confirmed working on dev. Closes #286

screen shot 2015-12-07 at 11 57 19 am

Not really sure how to test this as we don't have enough data to populate any of these facets. Suggestions?

### CODE REVIEW

**Comment by atz** [comment]
This was proposed and explicitly rejected previously.

**Comment by mejackreed** [comment]
Yet kind of accepted here: https://github.com/sul-dlss/argo/issues/286#issuecomment-162738983

**Comment by mejackreed** [comment]
Like I said, I'm happy to look into showing them all in more modal

**Comment by atz** [comment]
Accepted there because of a misimpression of technical difficulty.  It is technically trivial, you just don't like it.

**Comment by drh-stanford** [comment]
To test it we might be able to add lots (>100) of subject data (SW Topic or Genre) to a single fixture.

**Comment by mejackreed** [comment]
Trivial technical implementation that adds performance overhead to an already lagging application IMO isn't an actual solution.

**Comment by atz** [comment]
It doesn't add it, relative to production released Argo.  It merely maintains it.

**Comment by atz** [comment]
To be clear, I don't like it either and the presentation and runtime logic should be overhauled to keep the initial facet sidebar clean(er) and allow some as yet undetermined comprehensive modal/lookahead/filtering, etc.  I've had thoughts about how to accomplish that performantly, including caching the top level full facet values (in JSON) on the browser side.  But I don't see that really fitting into scope for this sprint.

**Comment by mejackreed** [comment]
:ok:

**Comment by mejackreed** [comment]
This PR is modified to accommodate temporary high limits for Collection, Admin Policy, Hydrus Collection, and Hydrus Admin Policy.

**Comment by drh-stanford** [comment]
i'm not able to test this out on my laptop due to limited data. we'll have to test this in `-dev`...

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -71,22 +71,22 @@ class CatalogController < ApplicationController
...
-    config.add_facet_field 'has_model_ssim',  :label => 'Object Model'
+    config.add_facet_field 'exploded_tag_ssim',               label: 'Tag',                 partial: 'blacklight/hierarchy/facet_hierarchy'
+    config.add_facet_field 'objectType_ssim',                 label: 'Object Type',         limit: 10
+    config.add_facet_field 'content_type_ssim',               label: 'Content Type',        limit: 10
+    config.add_facet_field 'rights_primary_ssi',              label: 'Access Rights',       limit: 10
+    config.add_facet_field 'use_license_machine_ssi',         label: 'License',             limit: 10
+    config.add_facet_field 'nonhydrus_collection_title_ssim', label: 'Collection',          limit: 9999, sort: 'index'
+    config.add_facet_field 'hydrus_collection_title_ssim',    label: 'Hydrus Collection',   limit: 9999, sort: 'index'
+    config.add_facet_field 'nonhydrus_apo_title_ssim',        label: 'Admin Policy',        limit: 9999, sort: 'index'
+    config.add_facet_field 'hydrus_apo_title_ssim',           label: 'Hydrus Admin Policy', limit: 10, sort: 'index'
```
Comment (jmartin-sul) [inline app/controllers/catalog_controller.rb]: `hydrus_apo_title_ssim` was supposed to get upped too, i think...

Comment (mejackreed) [inline app/controllers/catalog_controller.rb]: Good catch @jmartin-sul .. updated and pushed.

**Comment by jmartin-sul** [comment]
:+1:

**Comment by drh-stanford** [comment]
this is working in stage:

for the 9999 facets:

screen shot 2015-12-11 at 10 44 33 am


for the limited facets:
screen shot 2015-12-11 at 10 43 32 am

**Pull request merged**